lib/ansible/module_utils/homebrew.py

```diff
@@ -0,0 +1,245 @@
+# (c) 2013, Andrew Dunham <andrew@du.nham.ca>
+# (c) 2013, Daniel Jaouen <dcj24@cornell.edu>
+# (c) 2015-2017, Indrajit Raychaudhuri <irc+code@indrajit.com>
+#
+# This module is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
```

Comment:
Code in module_utils must not be under GNU GPL. Take BSD 2-clause please.

Reply:
This looks good to me. Note that the ```homebrew_cask``` module has to change as per: #31372